assets/app/views/directives/environment.html

```diff
@@ -0,0 +1,23 @@
+<div ng-if="envVars.length" class="table-responsive" style="margin-top: 5px;">
+  <table class="table table-bordered environment-variables">
+    <tbody>
+      <tr ng-repeat="env in envVars">
+        <td>{{env.name}}</td>
+        <td>
+          <!-- TODO we need to handle the valueFrom when it is defined instead of value -->
+          <span ng-if="!expanded[env.name] && (env.value | truncate : 200 : true : 3).length !== env.value.length">
+            <span class="env-var-value"><truncate-long-text content="env.value" limit="200" newline-limit="3"></truncate-long-text></span>
```

Comment:
Would this logic be simpler if we included an optional see all link in the truncate-long-text directive?